src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
+        super(color);
+        this.side = side;
+    }
+
+    public double getSide() {
+        return this.side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: " + getArea() + " sq. units, side: "
+                + this.side + " units, color: " + getColor());
```

Comment:
Use the `name()` method for getting the String representation of enum constants instead of `toString()` or `String.valueOf()` to ensure consistent behavior even if `toString()` is overridden in the enum.